src/database/migrations/3-create-products.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('products', {
+      id: {
+        allowNull: false,
+        autoIncrement: false,
```

Comment:
The idea was that a product can have its own ID to be set externally. It's when some resource / user get us list of products with full info (ID, name and so on). But yeah, in this case it rather better use standard incremental id field as it is, and another one (like productId) for the product itself (just in case).